setup.php

```diff
@@ -1289,14 +1289,31 @@ function cloneClicked() {
                         }
 
                         $short_tag = ini_get('short_open_tag') ? 'On' : 'Off';
+                        $short_tag_style = $short_tag.strcmp('Off') ? '' : 'text-danger';
```

Comment:
hi @stu01509 , The use of strcmp php function here doesn't look right:
https://www.php.net/manual/en/function.strcmp.php
think it should be:
```
$short_tag_style = strcmp($short_tag, 'Off') ? '' : 'text-danger';
```
(this will also make travis go away)

Reply:
Hello @bradymiller 

Thanks for your notice, It is very helpful.